frontend/.env.local.example

```
@@ -0,0 +1,8 @@
+# Backend API base URL.
+# In development, this proxies /api/* → http://localhost:8000 via next.config.ts rewrites.
+# In production, the rewrite in next.config.ts handles routing automatically.
+NEXT_PUBLIC_API_BASE_URL=http://localhost:8000
+
+# 32-character hex string used to AES-encrypt auth tokens in localStorage.
+# Generate with: node -e "console.log(require('crypto').randomBytes(16).toString('hex'))"
+NEXT_PUBLIC_AUTH_SECRET=<random 32-char hex string for AES encryption>
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)